### ISSUE TITLE
 http://rfidiot.org/ Down?

### ISSUE THREAD

**Issue by HeIp** [AdamLaurie/RFIDler#45]
[http://rfidiot.org/](http://rfidiot.org/#Hardware) website seems to not be working ?

**Comment by SteveClement** [comment]
Indeed :( @AdamLaurie

**Comment by AdamLaurie** [comment]
Yes - I'm working on it. My service provider had a major outage and lost
the VM it was running on :(

Thanks for the head's up!

Hope you're well!

cheers,
Adam
